src/resources/parser/glb-parser.js

```diff
@@ -98,18 +98,31 @@ Object.assign(pc, function () {
         return dummyIndices;
     };
 
-    var createVertexBuffer = function (device, attributes, indices, accessors, bufferViews, buffers) {
-        var semanticMap = {
-            'POSITION': pc.SEMANTIC_POSITION,
-            'NORMAL': pc.SEMANTIC_NORMAL,
-            'TANGENT': pc.SEMANTIC_TANGENT,
-            'BINORMAL': pc.SEMANTIC_BINORMAL,
-            'COLOR_0': pc.SEMANTIC_COLOR,
-            'JOINTS_0': pc.SEMANTIC_BLENDINDICES,
-            'WEIGHTS_0': pc.SEMANTIC_BLENDWEIGHT,
-            'TEXCOORD_0': pc.SEMANTIC_TEXCOORD0,
-            'TEXCOORD_1': pc.SEMANTIC_TEXCOORD1
+    var generateNormals = function (sourceDesc, vertexDesc, positions, numVertices, indices) {
+
+        if (!indices) {
+            indices = generateIndices(numVertices);
+        }
+
+        // generate normals
+        var normals = Float32Array.from(pc.calculateNormals(positions, indices));
```

Comment:
Sorry. This doesn't work in IE11: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypedArray/from

Reply:
I'll replace this .. this is existing code in main, so perhaps we wait till fix is in.

Reply:
fixed